SharedControls/Viewers/BuildingViewer.xaml

```diff
@@ -7,23 +7,55 @@
         xmlns:local="clr-namespace:Shared.Viewers"
         mc:Ignorable="d"
         WindowState="Maximized"
-        Title="{x:Static shared:GgLocalization.Task}" Height="450" Width="800">
-    <Grid>
+        MinHeight="1200"
+        MinWidth="1200"
+    Title="{x:Static shared:GgLocalization.Task}">
 
-        <Grid Margin="10">
-            <Grid.RowDefinitions>
-                <RowDefinition Height="Auto"/>
-                <RowDefinition Height="*"/>
-                <RowDefinition Height="Auto"/>
-            </Grid.RowDefinitions>
+    <StackPanel>
+        
 
-            <TextBlock Grid.Row="0" Text="{Binding TaskTitle}" Style="{StaticResource LargeHeaderStyle}"/>
+        <!-- Заголовок задания -->
+        <TextBlock
+                   Text="{Binding TaskTitle}" 
+                   Style="{StaticResource LargeHeaderStyle}" 
+                   HorizontalAlignment="Center"
+                   FontSize="36"
+                   Margin="20"/>
 
-            <StackPanel Grid.Row="1" x:Name="spItems" Margin="0 10">
-                <!-- Dynamic content will be added here -->
-            </StackPanel>
-
-            <Button Grid.Row="2" Content="{x:Static shared:GgLocalization.Check}" Click="btnCheck_Click" HorizontalAlignment="Right" Margin="10"/>
-        </Grid>
-    </Grid>
+        <!-- Контент задания (предложение и варианты) -->
+        <StackPanel
+                    x:Name="spItems" 
+                    VerticalAlignment="Center" 
+                    HorizontalAlignment="Center"
+                    Width="1200" 
+                    Height="600"
+                    Margin="0 300 0 0">
```

Comment:
Remove the **Margin** and **Height** from here. I couldn't see that button because of that margin, you should always try to make designs adaptive to various sizes and resolutions.